Translations/Translations.csv

```diff
@@ -0,0 +1,88 @@
+"no_subscriptions_error","You are not subscribed to any lists, foemp!","Je bent nog niet aangemeld bij een lijst, foemp!"
+"notify_add_list_usage","<list name>","<lijst naam>"
+"notify_add_list_brief","**admin-only** \n \u2003 Add a new list.","**admin-functie** \n \u2003 Voeg een nieuwe lijst toe."
+"notify_add_list_help","*admin-only* \u2003 Add a new list.\nYou will be asked what emoji to use for this list. React to the question of the bot with an emoji that is not yet used for another list. \n\n <list name> \u2003 The name of the list to add.","*admin-functie* \u2003 Voeg een nieuwe lijst toe.\nJe zal gevraagd worden om een emoji toe te voegen. Reageer op de vraag met een emoji die nog niet voor een andere lijst gebruikt wordt.\n\n<lijst naam> \u2003 De naam van de lijst om toe te voegen."
+"not_admin_gif","https://media.giphy.com/media/26gBjplxRU5EUhvlS/giphy.gif","https://gph.is/g/4w8PDNj"
+"list_already_exists","{0} already exists, foemp!","{0} bestaat al, foemp!"
+"list_emoji_request","What emoji do you want to use for {0}?","Welke emoji zou je willen gebruiken voor {0}?"
+"unknown_emoji","Emoji not recognized. Try again with a standard emoji or a custom emoji from this server, foemp!","De emoji kon niet herkend worden. Probeer opnieuw met een standaard emoji, of een emoji van deze server, foemp!"
+"emoji_already_in_use","This emoji is already used for a list, foemp!","Deze emoji is al in gebruik voor een lijst, foemp!"
+"add_list_success","The list {0} was saved with emoji {1}!","De lijst {0} werd opgeslagen met emoji {1}!"
+"notify_remove_list_usage","<list name>","<lijst naam>"
+"notify_remove_list_brief","**admin-only**\n\u2003 Remove a list.","**admin-functie**\n\u2003 Verwijder een lijst."
+"notify_remove_list_help","*admin-only* \u2003 Remove a list.\n\n<list name> \u2003 The name of the list to remove.","*admin-functie* \u2003 Verwijder een lijst.\n\n<lijst naam> \u2003 De naam van de lijst om te verwijderen."
+"confirmation_question","Are you sure?","Ben je zeker?"
+"remove_list_success","The list {0} was removed.","De lijst {0} is verwijderd."
+"remove_list_cancel","{0} won't be removed.","{0} wordt niet verwijderd."
+"snooze_lose","You snooze, you lose!","Te traag, foemp!"
+"settings_add_admin_usage","<mentioned user>","<gebruiker mention>"
+"settings_add_admin_brief","**admin-only**\n\u2003 Add a bot admin.","**admin-functie**\n\u2003 Voeg een bot admin toe."
+"settings_add_admin_help","*admin-only* \u2003 Add a bot admin. The user will be given access to the bot commands that are labelled **admin-only**. The user will not be given the role of server admin. \n\n<mention user> \u2003 Mention the user that you want to make a bot admin.","*admin-functie* \u2003 Voeg een bot admin toe. De gebruiker zal toegang krijgen tot de **admin funties** van de bot, maar geen extra rechten op de discord server. \n\n<gebruiker> \u2003 Link de gebruiker die je admin wilt maken."
+"settings_remove_admin_usage","<mentioned user>","<gebruiker>"
```

Comment:
Dutch key should be "getagde gebruiker" or "gebruiker mention" (whatever you decide for line 61)